### PR TITLE
Consolidate survey messages and use final content

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
## Description

    Describe your changes

## Reference

    i.e Jira or Github issue URL

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: addons/message_survey/OpenSurveyURL.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+((vpn) => {
+  // TODO once OS is available to addons
+  const desktop = true;
```
Comment (bakulf) [inline addons/message_survey/OpenSurveyURL.js:3]: To be tested:
`const desktop = [ "linux", "macos", "windows"].contains(api.env.platform);`

File: addons/message_survey/SetConditions.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+((vpn, condition) => {
```
Comment (bakulf) [inline addons/message_survey/SetConditions.js:1]: this can be removed.

File: addons/message_survey/manifest.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "api_version": "0.1",
+  "id": "message_survey",
+  "name": "Survey Message",
+  "type": "message",
+  "conditions": {
+    "trigger_time": 2419200,
+    "javascript": "SetConditions.js"
```
Comment (bakulf) [inline addons/message_survey/manifest.json:8]: I think we don't need the javascript condition if we restore the staging_1h and staging_30m

**Review by bakulf — CHANGES_REQUESTED** [review]
I would keep the `message_survey_staging_1h` and `message_survey_staging_30m`.
Doing this, we can have 1 prod survey message (message_survey) and 2 for testing.

File: addons/message_survey/manifest.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "api_version": "0.1",
+  "id": "message_survey",
+  "name": "Survey Message",
+  "type": "message",
+  "conditions": {
+    "trigger_time": 2419200
+  },
+  "message": {
```
Comment (lesleyjanenorton) [inline addons/message_survey/manifest.json:9]: @flodolo Content is final in this file only

**Review by flodolo — APPROVED** [review]

**Review by bakulf — APPROVED** [review]

**Pull request merged**